alpha/engagements/2024/Rust Foundation/update-2024-07.md

```diff
@@ -0,0 +1,48 @@
+There has been continued progress on verifying the repos that crates say they are associated with. We are also nearly complete on moving our security to the cloud in order to have more consistent runs, analysis and data.
+
+## Engineering
+
+### Crate repo verification progress
+
+Adam has continued working on[provenance](https://lawngno.me/blog/2024/06/10/divine-provenance.html) tracking, verifying that a given crate is actually associated with the repository it claims to be. He has now analyzed the top 5000 crates by download count and has found no obvious red flags.Verification now infers commits based on tag data if the crate file was packaged without VCS info. This finds more mismatches as a result, but they’re all innocent — build artifacts, swap files, etc. Up next is to analyze the entire crates.io corpus.
+
+### Crate deletion RFC
+
+The crate deletion [RFC](https://github.com/rust-lang/rfcs/pull/3660) is now in FCP. Recall, Tobias wrote this RFC to provide crate owners with a mechanism to delete crates from crates.io under certain [conditions](https://github.com/Turbo87/rust-rfcs/blob/crates-io/crate-deletions/text/3660-crates-io-crate-deletions.md#proposal).
```

Comment:
the RFC is not in FCP yet. I've proposed FCP, but the other team members still need to take a look and agree...

Reply:
```suggestion
The crate deletion [RFC](https://github.com/rust-lang/rfcs/pull/3660) is close to moving into FCP now. Recall, Tobias wrote this RFC to provide crate owners with a mechanism to delete crates from crates.io under certain [conditions](https://github.com/Turbo87/rust-rfcs/blob/crates-io/crate-deletions/text/3660-crates-io-crate-deletions.md#proposal).
```